src/main/java/com/worldwarofants/game/arch/AbstractController.java

```diff
@@ -21,7 +21,7 @@
  */
 public abstract class AbstractController<View extends AbstractView> {
 
-    private final IModuleNavigator moduleNavigator;
+    protected final IModuleNavigator moduleNavigator;
```

Comment:
What is the purpose of this change? It's not necessary. The way you use this is through the controller's method `navigateTo`. 

Which just now did I realize that there might be outdated documentation about that.. Kinda forgot I added that method here.. :D In my opinion, the code should stay, the docs should change.

Can you check the docs about it? If there are mistakes, make a new branch and a new PR for them, with the corrections. Because it's a separate task from the current one.

Reply:
No the documentation is correct. I thought I tried just using navigateTo() but it didn't work so I went through moduleNavigator, at which point I had to change its access. I'll change all the controllers to fix this.